### PR TITLE
feat: add lung v1 atlas and associated data (#6)

### PR DESCRIPTION
**Pull request by hunterckx**
Note: The project links (in the source datasets list) are currently hardcoded to point to the production Data Explorer (see [this line](https://github.com/clevercanary/hca-atlas-tracker/blob/a908396c7920684123b55da0af02d8e90fb365c1/app/viewModelBuilders/catalog/hca-atlas-tracker/common/viewModelBuilders.ts#L180)). This differs from the Data Portal, where it's based on the browserURL property in the config; here, browserURL is currently set to the Atlas Tracker's URL, although from what I can tell it's not actually used for anything

### CODE REVIEW

**Comment by NoopDog** [comment]
Thanks @hunterckx this looks great. @frano-m can you give this awesome effort the review it deserves? Thanks!

File: app/apis/catalog/hca-atlas-tracker/common/entities.ts
Comment (frano-m) [inline app/apis/catalog/hca-atlas-tracker/common/entities.ts:1]: Excellent naming of interface properties @hunterckx, thank you!

File: app/viewModelBuilders/catalog/hca-atlas-tracker/common/viewModelBuilders.ts [hunk 1]
```diff
@@ -20,20 +41,117 @@ export const buildAtlas = (
...
+/**
+ * Build props for the cell count cell component.
+ * @param componentAtlas - Component atlas entity.
+ * @returns Props to be used for the cell.
+ */
+export const buildCellCount = (
+  componentAtlas: HCAAtlasTrackerComponentAtlas
+): React.ComponentProps<typeof C.Cell> => {
+  return {
+    value: formatCountSize(componentAtlas.cellCount),
```
Comment (frano-m) [inline app/viewModelBuilders/catalog/hca-atlas-tracker/common/viewModelBuilders.ts:84]: Let's use `formatCountSize` from https://github.com/clevercanary/data-explorer/blob/b9e452696bf0d5e11eca6533d261fca4c681a5b3/packages/data-explorer-ui/src/utils/formatCountSize.ts. I may have added it to the findable package after this piece of code was added to the Data Portal.

File: app/viewModelBuilders/catalog/hca-atlas-tracker/common/viewModelBuilders.ts [hunk 2]
```diff
@@ -20,20 +41,117 @@ export const buildAtlas = (
...
+/**
+ * Build props for the estimated cell count cell component.
+ * @param sourceDataset - Source dataset entity.
+ * @returns Props to be used for the cell.
+ */
+export const buildEstimatedCellCount = (
+  sourceDataset: HCAAtlasTrackerSourceDataset
+): React.ComponentProps<typeof C.Cell> => {
+  return {
+    value: sourceDataset.estimatedCellCount?.toLocaleString(),
```
Comment (frano-m) [inline app/viewModelBuilders/catalog/hca-atlas-tracker/common/viewModelBuilders.ts:154]: Reminder: we may need to add in the additional code, if the estimated cell count is not available, to roll up the total cells `totalCells` from `cellSuspensions` see [here](https://github.com/DataBiosphere/data-portal/blob/984adb12724d18dcc66abf5b7951ccdce13108b5/next/viewModelBuilders/viewModelBuilders.ts#L70).

File: app/viewModelBuilders/catalog/hca-atlas-tracker/common/viewModelBuilders.ts [hunk 3]
```diff
@@ -50,6 +168,34 @@ export const buildIntegrationLead = (
...
+ * Build props for the project title cell component.
+ * @param sourceDataset - Source dataset entity.
+ * @returns Props to be used for the cell.
+ */
+export const buildProjectTitle = (
+  sourceDataset: HCAAtlasTrackerSourceDataset
+): React.ComponentProps<typeof C.Link> => {
+  return {
+    label: sourceDataset.projectTitle,
+    url: `https://explore.data.humancellatlas.org/projects/${sourceDataset.projectId}`, // TODO different source for base URL?
```
Comment (frano-m) [inline app/viewModelBuilders/catalog/hca-atlas-tracker/common/viewModelBuilders.ts:195]: Reminder: we will probably need to add back in cases for when the source is external (i.e. the source is not HCA), or when the source is unpublished. See [here](https://github.com/DataBiosphere/data-portal/blob/main/next/viewModelBuilders/viewModelBuilders.ts#L462).
For example: 
- `Schiller_2021` is currently unpublished, and
- `Expansion of profibrotic monocyte-derived alveolar macrophages in patients with persistent respiratory symptoms and radiographic abnormalities after COVID-19` is sourced externally at https://www.biorxiv.org/content/10.1101/2023.07.30.551145v1.

Comment (frano-m) [reply app/viewModelBuilders/catalog/hca-atlas-tracker/common/viewModelBuilders.ts:195]: Let's discuss `baseURL` at a later state 😄

File: app/viewModelBuilders/catalog/hca-atlas-tracker/common/viewModelBuilders.ts [hunk 4]
```diff
@@ -111,3 +285,63 @@ export function getBioNetworkByKey(key: NetworkKey): Network | undefined {
...
+): string {
+  return PLURALIZED_METADATA_LABEL[metadataKey];
+}
+
+/**
+ * Formats count sizes.
+ * @param value - Count size.
+ * @returns formatted count size as display string.
+ */
+export function formatCountSize(value: number): string {
```
Comment (frano-m) [inline app/viewModelBuilders/catalog/hca-atlas-tracker/common/viewModelBuilders.ts:305]: We can remove the function `formatCountSize` if we reference the findable library.

File: app/viewModelBuilders/catalog/hca-atlas-tracker/common/viewModelBuilders.ts (hunk 2 above)
Comment (frano-m) [reply app/viewModelBuilders/catalog/hca-atlas-tracker/common/viewModelBuilders.ts:154]: Ignore: thank you @hunterckx, I see this is taken care of by: https://github.com/clevercanary/hca-atlas-tracker/pull/12/files#diff-24cafcee291461cbfb84b7615792be3ca6b645bd0be88b7632e2d1d52d556779R151.

**Review by frano-m — APPROVED** [review]
Outstanding @hunterckx.

I've added a few reminders for us to circle back at some point and resolve. This is excellent work. Thank you.

**Pull request merged**